File: modul_praktikum_python/contoh_code/pascal/pangkat_for.pas
uses crt; 
var 
  i,pangkat:integer;   
  hasilpangkat,dipangkatkan:real; 
begin   
 write(‘Bilangan yang akan dipangkatkan : ‘); 
 readln(dipangkatkan); 
 write(‘Masukkan pangkatnya : ‘);readln(pangkat); 
 hasilpangkat:=1; 
 for i:=1 to pangkat do  
    hasilpangkat:=hasilpangkat*dipangkatkan; 
 write(dipangkatkan,‘^’,pangkat,’ : ‘,hasilpangkat); 
readkey; 
end.

File: modul_praktikum_python/contoh_code/python/pangkat_for.py
"""Pangkat bilangan real dengan perulangan for."""


def main() -> None:
    dipangkatkan = float(input("Bilangan yang akan dipangkatkan : "))
    pangkat = int(input("Masukkan pangkatnya : "))
    hasilpangkat = 1.0
    for _i in range(1, pangkat + 1):
        hasilpangkat *= dipangkatkan
    print(dipangkatkan, "^", pangkat, " : ", hasilpangkat, sep="")
    input("Tekan Enter untuk melanjutkan...")


if __name__ == "__main__":
    main()
